sw/airborne/firmwares/rotorcraft/main_ap.c

```diff
@@ -317,6 +317,9 @@ void failsafe_check(void)
       autopilot_get_motors_on() &&
 #if NO_GPS_LOST_WITH_RC_VALID
       radio_control.status != RC_OK &&
+#endif
+#if NO_GPS_LOST_WITH_DATALINK_TIME
```

Comment:
it is more correct to use `#ifdef` for non-boolean values, although it makes no real difference here